lib/ansible/modules/cloud/vmware/vmware_guest_find.py

```diff
@@ -28,22 +28,22 @@
     - PyVmomi
 options:
    name:
-        description:
-            - Name of the VM to work with.
-            - This is required if uuid is not supplied.
+     description:
+     - Name of the VM to work with.
+     - This is required if C(uuid) parameter is not supplied.
```

Comment:
Do we also want `mutually_exclusive=(['name', 'uuid'],),`?

Reply:
Done.